source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/WebApi/ModelPredicates/ChargePointQueryLogicTests.cs

```diff
@@ -0,0 +1,271 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Energinet.DataHub.Charges.Contracts.Charge;
+using Energinet.DataHub.Charges.Contracts.ChargePrice;
+using Energinet.DataHub.Core.TestCommon.AutoFixture.Attributes;
+using FluentAssertions;
+using GreenEnergyHub.Charges.QueryApi.Model;
+using GreenEnergyHub.Charges.QueryApi.ModelPredicates;
+using GreenEnergyHub.Iso8601;
+using Xunit;
+using Xunit.Categories;
+
+namespace GreenEnergyHub.Charges.Tests.WebApi.ModelPredicates
+{
+    [UnitTest]
+    public class ChargePointQueryLogicTests
+    {
+        [Theory]
+        [InlineAutoMoqData]
+        public void AsChargePointV1Dto_WhenResolutionIsPT15M_SetsAllPropertiesCorrectly(
```

Comment:
Below suggestion goes for all tests

```suggestion
        public void AsChargePriceV1Dto_WhenResolutionIsPT15M_SetsAllPropertiesCorrectly(
```